.storybook/components/ColorList/ColorList.module.css

```diff
@@ -1,6 +1,6 @@
 .color-list__label {
   width: fit-content;
-  padding: calc(var(--eds-spacing-size-half) / 16 * 1rem);
+  padding: calc(var(--eds-spacing-size-half) * 1px);
```

Comment:
with `padding`, `margin`, and other size-like fields, we should use pixel conversions